tools/bigquery-to-xml/bq_to_xml.py

```diff
@@ -15,9 +15,9 @@
 #   limitations under the License.
 
 import os
-import xml.etree.ElementTree as ET
+from xml.etree import ElementTree
```

Comment:
Ik this is a pain but this still does not conform to the style guide which says to import modules not just individual classes / methods.

the idea is to make it very explicit where a method or class comes from in the code w/o having to look back at the import statements. In your case you actually have two different etree modules (form xml and lxml) which actually really drives home the point to be very explicit.
```suggestion
import xml.etree
```